internal/declarative/v2/reconciler.go

```diff
@@ -450,11 +466,6 @@ func (r *Reconciler) pruneDiff(
 	if err != nil {
 		return err
 	}
-	resourceName := r.ModuleCRDName(obj)
```

Comment:
do we still have usage for this function after removing it here? If not, we can cleanup completely.

Reply:
Good catch! Will clean it up, not needed anymore :D